crates/nu-std/lib/help.nu

```diff
@@ -671,13 +672,19 @@ export def "help commands" [
             $found_commands | select name category usage signatures search_terms
         }
     } else if not ($command | is-empty) {
-        let found_command = ($commands | where name == $command)
+        let found_commands = ($commands | where name == $command)
 
-        if ($found_command | is-empty) {
-            command-not-found-error (metadata $command | get span)
+        if not ($found_commands | is-empty) {
+            show-command ($found_commands | get 0)
+        } else {
+            try {
+                print $"(ansi default_italic)Help pages from external command ($command | pretty-cmd):(ansi reset)"
+                 ^($env.NU_HELPER? | default "man") $command
+            } catch {
+                command-not-found-error (metadata $command | get span)
```

Comment:
I don't think that this should be a command-not-found error because it will also catch errors such as `man` not being installed. Removing this (the try catch) does not seem to work if you just run `help <external` and `man` (or your configured command) is not found

Reply:
currently in the std, if this errors when running `help <command>` (such as `help s`) it will show nothing. If you do `help commands s` it will error

Reply:
are you sure it does not work?
this `try`-`catch` block does not look that helpful here to me

- if the command is internal, we run `show-command`
- if the command is not internal, we run `NU_HELPER` and it's `NU_HELPER` which handles the case where the command does not have a `man` / `tldr` page

i think we do not even need the `command-not-found-error` error now because, as a command is either internal or extenal, we can't not find an error :smirk:

Reply:
but that can be done in another PR as well as the `--find` option, no worries :wink: